applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp

```diff
@@ -252,6 +252,22 @@ namespace Kratos
 
 		//FORCE- & Moment- Residual is 3D vector
 		KRATOS_TRY;
+
+		bounded_vector<double,msElementSize> damping_residual_contribution = ZeroVector(msElementSize);
+		// calculate damping contribution to residual -->
+		if((this->GetProperties().Has(RAYLEIGH_ALPHA) || this->GetProperties().Has(RAYLEIGH_BETA)) && (rDestinationVariable != NODAL_INERTIA))

```

Comment:
you are calling the Has method twice. Take care: it is rather slow. 

    bool has_rayleigh_alpha = this->GetProperties().Has(RAYLEIGH_ALPHA)
    if(has_rayleigh_alpha || ..... )